src/index.js

```diff
@@ -1,18 +1,13 @@
-/* eslint-disable import/no-cycle */
 import './css/styles.css';
-import leaderboardItems from './getItems.js';
+import getLeaderboardItems from './getItems.js';
+import addLeaderboardItems from './addItem.js';
 
-const getLeaderboardItem = () => {
-  leaderboardItems.forEach((leaderboardItem) => {
-    const list = document.getElementById('leaderboard-list');
-    const listLi = document.createElement('tr');
-    listLi.innerHTML = `
-     <td>${leaderboardItem.name}:</td>
-     <td>${leaderboardItem.score}</td>
-    `;
+window.onload = async () => {
+  const submitItem = document.getElementById('submitButton');
+  const refreshItem = document.getElementById('refresh');
 
-    list.appendChild(listLi);
-  });
-};
+  getLeaderboardItems();
 
-getLeaderboardItem();
\ No newline at end of file
+  submitItem.addEventListener('click', addLeaderboardItems);
```

Comment:
[OPTIONAL] Since you used correctly the `form` tag, you could also listen for the `submit` event. This one is triggered not only when the `input type=submit` is clicked, but also when the user hits the letter enter. Unfortunately, JS will try to use an action and a method for the form, redirecting the page elsewhere.

To avoid the last, we need to prevent the default behavior of the form document, you can do this like this:
```
form.addEventListener('submit', (event) => {
  event.preventDefault();
  addLeaderboardItems();
});
```

I hope you find this insightful :sparkles:

Reply:
Thank you for the recommendation. They are great for implementation